torch/_prims/nvfuser_executor.py

```diff
@@ -196,7 +211,7 @@ def nvfuser_execute(gm: GraphModule, *args, executor_parameters=None):
         )
 
         return tree_unflatten(
-            fusion.execute(concrete_fusion_inputs),  # type: ignore[has-type]
+            fusion.execute(concrete_fusion_inputs)[drop_output_count:],  # type: ignore[has-type]
```

Comment:
are the implicit extra outputs guaranteed to be in the beginning?

Reply:
Yes, the order of adding outputs is respected and the real outputs we need to return are added last here:
https://github.com/pytorch/pytorch/blob/705b2b275618235bc7c670d5161d4de82f420ca7/torch/_prims/nvfuser_executor.py#L175-L176